packages/website/components/layout/index.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import clsx from 'clsx';
+
+import GradientBackground from '../gradientbackground/gradientbackground';
+import TextBlock from '../textblock/textblock';
+
+/**
+ *
+ * @param {Object} props
+ * @param {React.ReactNode} props.children
+ * @param {string} [props.variant]
+ * @param {string} [props.id]
+ *
+ * @returns
+ */
+export function Sectionals({ children, variant, id }) {
```

Comment:
The existing code used the name "sectionals," and then each of these contained a number of "sectional" components, and within "sectional" you'll find "column."  The naming needs a little help, I think, and I'm open to ideas.  Maybe "Container" for the outermost box, then "Section," and then "Column"?